metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/RemoveUnknownAspectsStep.java

```diff
@@ -0,0 +1,76 @@
+package com.linkedin.metadata.boot.steps;
+
+import com.linkedin.common.AuditStamp;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.events.metadata.ChangeType;
+import com.linkedin.gms.factory.telemetry.TelemetryUtils;
+import com.linkedin.metadata.Constants;
+import com.linkedin.metadata.boot.BootstrapStep;
+import com.linkedin.metadata.entity.EntityService;
+import com.linkedin.metadata.key.DataHubUpgradeKey;
+import com.linkedin.metadata.utils.EntityKeyUtils;
+import com.linkedin.metadata.utils.GenericRecordUtils;
+import com.linkedin.mxe.MetadataChangeProposal;
+import com.linkedin.upgrade.DataHubUpgradeResult;
+import java.util.HashMap;
+import javax.annotation.Nonnull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@RequiredArgsConstructor
+public class RemoveUnknownAspectsStep implements BootstrapStep {
```

Comment:
Do we need both an upgrade step and bootstrap step?

Reply:
Having both allows people to get by this issue when stuck on a previous release